Resources/Prototypes/_LostParadise/Entities/Clothing/Eyes/Glasses/glasses.yml

```
@@ -77,3 +77,17 @@
   - type: FlashImmunity
   - type: EyeProtection
     protectionTime: 5
+
+- type: entity
+  parent: [ClothingEyesBase, ShowSecurityIcons]
+  id: LPPClothingEyesGlassesAviatorNPZ
+  name: NPZ aviators
+  description: Only for the most coolest guys in this sector.
+  components:
+  - type: Sprite
```

yamllint

[error] wrong indentation: expected at least 3

(indentation)